packages/dids/package.json

```diff
@@ -76,41 +76,36 @@
   "dependencies": {
     "@decentralized-identity/ion-pow-sdk": "1.0.17",
     "@decentralized-identity/ion-sdk": "1.0.1",
+    "@noble/hashes": "^1.3.2",
+    "bittorrent-dht": "^11.0.5",
```

Comment:
@decentralgabe any particular reason to list `bittorrent-dht` as an explicit dependency?  It appears to be a transitive dependency of the `pkarr` package.  Could lead to version conflicts to have it specified in `@web5/dids`.

Reply:
remnant, removing